src/script/lua_api/l_mainmenu.cpp

```diff
@@ -844,22 +844,21 @@ bool ModApiMainMenu::isMinetestPath(std::string path)
 		return true;
 
 	/* games */
-	if (fs::PathStartsWith(path,fs::RemoveRelativePathComponents(porting::path_share + DIR_DELIM + "games")))
+	if (fs::PathStartsWith(path,fs::RemoveRelativePathComponents(porting::path_user + DIR_DELIM + "games")))
```

Comment:
path_share is also a valid place for it to be, so I suggest duplicating this statement - one for share and one for user

Reply:
I thought that method was to allow downloading to user directories only?

Reply:
It is indeed only used to check whether the directory may be modified. However, I suggest renaming this function to `mayModifyPath` or similar.

Reply:
It'd be nice if you could add a space after `path,`. Many such spaces are missing throughout the file, but at least the code you touched can be rid of code-style issues.